src/itty-fetcher.ts

```diff
@@ -39,14 +39,24 @@ const fetchy = (options: FetchyOptions): FetchyFunction => (
   payload?: string | number | object | undefined,
   fetchOptions?: FetchOptions,
 ) => {
-  const {
-    base,
-    autoParse,
-    method,
-  } = options
+  const method = options.method.toUpperCase()
+  const resolvedURL = new URL(options.base + url)
```

Comment:
I think the `+` is likely to be contentious or need documentation...

Reply:
That was already existing. I would prefer to do a more intelligent concatenation tho

Reply:
This is a reference to #2 not your fault.